### PR TITLE
ruth brand final

### PR DESCRIPTION
**Pull request by RuthBrand**
you guys, its SO gross and i apologize, but it works all the way through iteration three. if you want to see my thought process you can see it throughout the commits as i wrote a lot down. there is a specific commit that says "look at all commits prior to this to see more scratch work" - after that i decided i needed to move forward with speed and just do what i know, so i deleted a lot of comments. its VERY hack-y. BUT IT WORKS!!

### CODE REVIEW

**Comment by RuthBrand** [comment]
see above

File: lib/curator.rb [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    photo.artist_id == artist.id
+   end
+  end
+
+  def artists_with_multiple_photographs
+    ids = @photographs.map do |photo|
+      photo.artist_id
+    end
+
+    first_artist = find_artist_by_id(ids.first)
```
Comment (memcmahon) [inline lib/curator.rb]: Looking forward to seeing the refactor on this 😄

File: lib/curator.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    if count_of_third_artist > 1
+      artists_greater_than_one << third_artist
+    end
+
+    artists_greater_than_one
+  end
+
+
+  def photographs_taken_by_artists_from(location)
```
Comment (memcmahon) [inline lib/curator.rb]: It seems like this method, and the one above, will only work with three artists - what if we have 300 artists?

File: test/curator_test.rb [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+    }
+
+    photo_2_attributes = {
+      id: "2",
+      name: "Moonrise, Hernandez",
+      artist_id: "2",
+      year: "1941"
+    }
+
+    photo_1 = Photograph.new(photo_1_attributes)
```
Comment (memcmahon) [inline test/curator_test.rb]: per the spec, you should be creating the Photograph instances within the Curator class!

**Pull request closed without merging**